locale/ara/LC_MESSAGES/django.po

```diff
@@ -51983,7 +51983,7 @@ msgstr "الموقع الأصل"
 
 #: corehq/ex-submodules/couchforms/openrosa_response.py
 msgid "   √   "
-msgstr "      "
+msgstr ""
```

Comment:
I'd have thought this would be translated into a checkmark, the same as in English, although I don't know that I have a good reason for that. Why is it blank?

Reply:
An empty `msgstr` should mean it just uses the default `"   √   "` rather than assigning a language-specific translation. This is what it looks like in existing files for other languages as well, e.g.:
https://github.com/dimagi/commcare-hq/blob/60d1bf1b53970d1294a70a12ad019c3e3a949259/locale/es/LC_MESSAGES/django.po#L45702-L45704